openshift/ci-operator/knative-images/gcs-fetcher/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# Do not edit! This file was generated via Makefile
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+ADD gcs-fetcher /ko-app/gcs-fetcher
```

Comment:
where is the `gcs-fetcher` stored? How does this get downloaded?

Reply:
ci-operator builds it
1. https://github.com/openshift/release/blob/610fc91fd0c26eb1d84a458b80ff77805c4f10b8/ci-operator/config/openshift/tektoncd-pipeline/openshift-tektoncd-pipeline-release-v0.7.0.yaml#L16
1. https://github.com/openshift/tektoncd-pipeline/blob/953d50db65a8e4967b9a848952e1756990681420/Makefile#L21

so it will be copied like the other binaries from `./cmd`

Reply:
thank you 🤗